recipe/meta.yaml

```
@@ -20,13 +20,14 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - dataclasses
-    - jax >=0.2.6
+    - jax >=0.3.2
     - matplotlib-base
     - msgpack-python
     - numpy >=1.12
-    - optax >=0.0.6
+    - optax
+    - rich ~=11.1.0
```

```suggestion
    - rich >=11.1.0
```
Since conda does not work with upper bounds, let's not have them anywhere...